src/libutil/unix/environment-variables.cc

```diff
@@ -9,4 +9,14 @@ int setEnv(const char * name, const char * value)
     return ::setenv(name, value, 1);
 }
 
+std::optional<std::string> getEnvNative(const std::string & key)
+{
+    return getEnv(key);
+}
+
+int setEnvNative(const OsString & name, const OsString & value)
```

Comment:
Is it Native or Os? - could we pick one term?

Reply:
Fine either way, have a few other "natives" already but they are all new enough that I think we should pick the best name without historical precedent.

(N.B. this is part of the previous PR however.)

Reply:
OS is shorter and clearer IMO.

Reply:
Rename is done